src/main/webapp/projects.html

```diff
@@ -14,6 +14,9 @@
   <body>
     <div id="navbar-container"></div>
     <div id="projects-container"></div>
+    <noscript>
+      Please enable JavaScript to use this website.
```

Comment:
Q: Should we be adding this check to every page in the site?

Reply:
(I'm pretty sure almost nothing we have would work without js)

Reply:
Yeah most likely. I wanted the choice to ultimately reside in the "owner" of the page, so feel free to add this to your HTML pages as well.